src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -63,7 +63,8 @@ std::string postEntity
 )
 {
   Entity*        eP  = &parseDataP->ent.res;
-  std::string    op  = ciP->uriParam["op"];
+  //std::string    op  = ciP->uriParam["op"];
```

Comment:
Leftover?
Wait, is the `op` uri param no longer used?

Reply:
Yes, it's a leftover.

In fact, this was weird in the current code. `op` is set form a non-existing op= parameter (it was in very early NGSIv2 versions, but at the end we preferred options=append for the same)... to be overriden later. This PR fix that weird-ness.

Reply:
Fixed in 206e503